playbooks/ansible-test-network-integration-base/pre.yaml

```diff
@@ -36,6 +37,7 @@
         ansible_test_inventory_os: "{{ _network_os }}"
         ansible_test_inventory_dest: ~/inventory
         vmware_ci_set_passwords_secret_dir: '{{ zuul.executor.work_root }}'
+        trendmicro_deepsec_api_key: '{{ zuul.executor.work_root }}'
```

Comment:
same comment, seems unused

Reply:
Yes, the path is hardcoded in roles/ansible-test-inventory/templates/inventory-trendmicro.deepsec.deepsec.j2.

I suggest to rename this variable from `trendmicro_deepsec_api_key` to `trendmicro_deepsec_secret_dir` to be consistent with wmare. And off course, reuse the variable in `inventory-trendmicro.deepsec.deepsec.j2`.

Reply:
I would actually prefer to use one single variable called `ansible_test_inventory_secret_dir`.